api/agentpb/agent.proto

```diff
@@ -506,15 +506,15 @@ message StartJobRequest {
   // MongoDBBackup is job for backup MongoDB service.
   message MongoDBBackup {
     // Database user;
-    string user = 1;
+    string user = 1 [deprecated = true];
     // Database password.
-    string password = 2;
+    string password = 2 [deprecated = true];
     // Database address. Can't be specified with socket.
-    string address = 3;
+    string address = 3 [deprecated = true];
     // Database port. Can't be specified with socket.
-    int32 port = 4;
+    int32 port = 4 [deprecated = true];
     // Database unix socket. Can't be specified with address/port.
-    string socket = 5;
+    string socket = 5 [deprecated = true];
```

Comment:
I see we are deprecating a bunch of attributes. Q: does that have any effect on our API?

Reply:
It doesn't affect endpoints, it affects only server-agent communication.